src/python/pants/backend/python/tasks/isort_prep.py

```diff
@@ -55,7 +55,8 @@ def create_requirements(cls, context, workdir):
       def build_isort_pex(cls, context, interpreter, pex_path, requirements_lib):
         with safe_concurrent_creation(pex_path) as chroot:
           builder = PEXBuilder(path=chroot, interpreter=interpreter)
-          dump_requirement_libs(builder=builder,
+          pex_build_util = PexBuildUtil(PythonRepos.global_instance(), PythonSetup.global_instance())
```

Comment:
The `IsortPrep.subsystem_dependencies` override should be moved up here into `IsortPrep.Isort.Factory`.